pkg/Microsoft.Private.PackageBaseline/packageIndex.json

```diff
@@ -541,7 +628,8 @@
     "runtime.win7-x64.runtime.native.System.IO.Compression": {
       "StableVersions": [
         "4.0.1"
-      ]
+      ],
+      "BaselineVersion": "4.3.0"
     },
     "runtime.win7-x86.runtime.native.System.Data.SqlClient.sni": {
```

Comment:
Yes, any TFS-built package changes made on master since 1.0.0 won't be available through 1.1.0 packages with this plan.

@weshaggard @gkhanna79 Should we make a new TFS branch/build so 1.1.0 can take SNI changes? The plan to continue using 1.0.0 RTM versions was with the understanding that there weren't any fixes to take. (Email thread `1.1 bring up`)